sentry-android/src/main/java/io/sentry/android/AndroidSentryClientFactory.java

```diff
@@ -91,6 +98,34 @@ public SentryClient createSentryClient(Dsn dsn) {
         SentryClient sentryClient = super.createSentryClient(dsn);
         sentryClient.addBuilderHelper(new AndroidEventBuilderHelper(ctx));
 
+        boolean enableAnrTracking = "true".equalsIgnoreCase(Lookup.lookup("anr.enable", dsn));
+        Log.d(TAG, "ANR is='" + String.valueOf(enableAnrTracking) + "'");
+        if (enableAnrTracking && anrWatchDog == null) {
+            String timeIntervalMsConfig = Lookup.lookup("anr.timeoutIntervalMs", dsn);
+            int timeoutIntervalMs = timeIntervalMsConfig != null
+                    ? Integer.parseInt(timeIntervalMsConfig)
```

Comment:
Since this is user input would it make sense to catch `NumberFormatException`s and fallback to the default value?

Reply:
Honestly I'd rather let it bubble so the user can see he's misconfigured.